library/src/main/java/com/bumptech/glide/load/resource/gif/GifDrawable.java

```diff
@@ -134,6 +134,14 @@ public int getFrameCount() {
     return state.frameLoader.getFrameCount();
   }
 
+  /**
+   * Returns the current frame index in the range of 0 to {@link #getFrameCount()}-1.
```

Comment:
Urk, sorry I reworded my comment forgot to remove a critical word. Thanks, this is fine, I'll adjust the comment and merge, sorry for the confusion!